extensions/product-recs/content/src/main/content/jcr_root/apps/core/cif/extensions/product-recs/components/productrecommendations/v1/productrecommendations/clientlibs/editor/css/dialog.css

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************
+ *
+ *    Copyright 2021 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+.cif-product-recs__price-range {
+    margin-top: 0 !important;
```

Comment:
don't use `!importnat`, this will prevent any downstream consumer to overwrite the styling.

Reply:
this is dialog styling. is this ever overwritten ?

Reply:
If you want to get rid of the `!important` keyword, try to use a selector that is more specific than the one that comes from CoralUI which you would like to override.

Reply:
> is this ever overwritten ?

yes it is - even if not supposed to ;)